plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/ServerSettingsWizardPageModel.java

```diff
@@ -472,7 +472,7 @@ public void setRoute(IRoute newRoute) {
 				this.deployProject, this.projects, 
 				this.sourcePath, this.podPath, this.useInferredPodPath,
 				getService(), getServiceItems(),
-				route, this.selectDefaultRoute, this.routesByProject,
+				newRoute, this.selectDefaultRoute, this.routesByProject,
```

Comment:
currently writing tests for this model, and I also came across the fact that #getRoutes() returns all routes, while I keep wondering if we should only return the routes that match the currently selected service. getRoutes() is actually what the user gets as possible choices in the route combo. Does it makes sense that the user can choose whatever route he wants, even a route that's not related to the service that's currently selected?